lib/http/apps/migrate.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+
+const app = express();
+
+
+/**
+ * @api {post} /migrate/space
+ * @apiName PostMigrateSpace
+ * @apiGroup Migrate
+ * @apiVersion 1.0.0
+ * 
+ * @apiParam {Object[]} accounts List of accounts objects
+ *  
+ * @apiUse MissingAccountIdError
+ * @apiUse MissingDaoError
+ */
+app.post('/space', (req, res) => {
+    const accounts = req.body.accounts;
+  
+    if (!Array.isArray(accounts)) {
+        res.status(400).json({error: 'accounts is missing'});
+    } else {
+        accounts.forEach((account) => {
+            res.locals.coinsence.Space.functions.addMembers(account.address, { gasLimit: 5000000 })
```

Comment:
are we sure that the account is not yet a member? I guess it is ok for now.

Reply:
I think in the contracts if an account already a member, nothing happen...